apps/pom.xml

```diff
@@ -1025,10 +1025,15 @@
                     <failOnWarnings>false</failOnWarnings>
                     <detectJavaApiLink>false</detectJavaApiLink>
                     <!-- Exclude javadocs on certain generated files, reduces build warnings -->
-                    <excludePackageNames>gov.cms.mpsm.rda.*</excludePackageNames>
-                    <excludePackageNames>gov.cms.bfd.model.rif.entities.*</excludePackageNames>
-                    <excludePackageNames>gov.cms.bfd.model.rda.entities.*</excludePackageNames>
-                    <quiet>true</quiet>
+                    <sourceFileExcludes>
+                        <exclude>**/**_.java</exclude>
+                        <exclude>**/gov/cms/bfd/model/rda/entities/**.java</exclude>
+                        <exclude>**/gov/cms/bfd/model/rif/entities/**.java</exclude>
+                        <exclude>**/gov/cms/bfd/pipeline/rda/grpc/source/parsers/**.java</exclude>
+                        <exclude>**/gov/cms/bfd/pipeline/rda/grpc/server/RdaService.java</exclude>
+                        <exclude>**/gov/cms/mpsm/rda/**</exclude>
+                    </sourceFileExcludes>
+                    <quiet>false</quiet>
```

Comment:
Was this changed for debugging, or do we want this to not be quiet?

Reply:
yh. I'll change it back.